docs/api.rst

```diff
@@ -0,0 +1,47 @@
+Orcoursetrion API Docs
+----------------------
+
+For convenient reference in development, here are the Orcoursetrion
+API docs.
+
+Actions
+=======
+
+The actions that are available to use.
+
+.. automodule:: orcoursetrion.actions
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :imported-members:
+
+Library
+=======
+
+API libraries.
+
+.. automodule:: orcoursetrion.lib
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :imported-members:
+
+Configuration
+=============
+
+Configuration options
+
+.. automodule:: orcoursetrion.config
+
+.. autoattribute:: orcoursetrion.config.GH_API_KEY
+   :annotation: = GitHub API Key
```

Comment:
Maybe "Github API token"  if this is what github calls  "Personal access tokens" ?